src/client/public/auth.ts

```diff
@@ -76,12 +76,14 @@ function proceedWithAuthInit(authRequest: AuthRequestParamData) {
             if (hasHash) {
                 const authResponseKeyValues = Authenticator.extractParams(window.location.href.split('#')[1]);
                 const accessToken = authResponseKeyValues['access_token'];
+                const expiresIn = authResponseKeyValues['expires_in'];
                 if (accessToken) {
+                    const message = 'AUTH:access_token=' + accessToken + '&AUTH:expires_in=' + expiresIn;
```

Comment:
Do you want to make this into a template string?

    const message = `AUTH:access_token=${accessToken}&AUTH:expires_in${expiresIn};

Reply:
As far as the message goes, btw:  I was being super lazy last time, so hence I just did a key-value singleton.  But really it should just be a message, something like:

    {
        type: "auth",
        message: {
            accessToken: accessToken,
            expiresIn: expiresIn
        }
    }

Then stringify it and send it as a string.  And on the receiving end, check that the object is a string, do a try/catch around the JSON.stringify, and then do a check for whether the resulting object has a "type" property whose value is "auth".